src/stackmap_checker/utils.c

```diff
@@ -0,0 +1,26 @@
+#include <elf.h>
+#include <string.h>
+#include <unistd.h>
+#include <sys/mman.h>
+#include <fcntl.h>
+#include "utils.h"
+
+void *get_addr(const char *section_name)
```

Comment:
Gets the base address of this section?

Reply:
I've documented that in `utils.h`. Should I write the documentation in the `.c` files from now on?